src/test/java/org/opensearch/plugin/prometheus/PrometheusPluginIT.java

```diff
@@ -17,8 +17,9 @@
 package org.opensearch.plugin.prometheus;
 
 import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
-import org.apache.http.Header;
-import org.apache.http.util.EntityUtils;
+import org.apache.hc.core5.http.Header;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.apache.hc.core5.http.ParseException;
```

Comment:
Thanks, I think this was a major change in OS 3.0 related PR https://github.com/opensearch-project/security/pull/3741